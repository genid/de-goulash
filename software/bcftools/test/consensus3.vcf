##fileformat=VCFv4.2
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
##INFO=<ID=END,Number=1,Type=Integer,Description="End position of the variant described in this record">
##ALT=<ID=DEL,Description="Deletion">
##reference=file://some/path/human_g1k_v37.fasta
##contig=<ID=1,assembly=b37,length=249250621>
##contig=<ID=2,assembly=b37,length=249250621>
##contig=<ID=3,assembly=b37,length=249250621>
##contig=<ID=4,assembly=b37,length=249250621>
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	NA001
1	1	.	C	.	.	PASS	.	GT	.
1	2	.	T	G	.	PASS	.	GT	.
1	10	.	GTGA	G	.	PASS	.	GT	0|1
1	13	.	A	ATT	.	PASS	.	GT	0|1
