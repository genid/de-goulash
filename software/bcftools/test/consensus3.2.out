>xx_1
??ACCATATGTTCATATAAAAAAGAACATAACCTACGTATCAACTAAAGTGGTTGTTC
>xx_2
CCTACCATATGTGACATATAAAAAAGAACATAACCTACGTATCAACTAAAGTGGTTGTTC
>xx_3
CCCTACCATATGTGACATATAAAAAAGAACATAACCTACGTATCAACTAAAGTGGTTGTT
C
>xx_4
CCCCTACCATATGTGACATATAAAAAAGAACATAACCTACGTATCAACTAAAGTGGTTGT
TC
